src/CSTools/Tools/parser.cs

```diff
@@ -1693,7 +1693,7 @@ public static object create(string cls_name,Parser yyp)
 				yyp.m_symbols.erh.Error(new CSToolsException(16,yyp.m_lexer,"no factory for {"+cls_name+")"));
 			try
 			{
-				return cr(yyp);
+				return cr!(yyp);
```

Comment:
here's the same as a previous comment, we should just skip this if it's null as the previous check may not throw